integration-tests/vertx-http/pom.xml

```diff
@@ -103,6 +103,7 @@
                                         <additionalBuildArg>-H:EnableURLProtocols=http,https</additionalBuildArg>
                                     </additionalBuildArgs>
                                     <graalvmHome>${graalvmHome}</graalvmHome>
+                                    <enableJni>true</enableJni>
```

Comment:
I don't understand that? I mean the extensions themselves should trigger the enablement of JNI, no?

Reply:
Same answer as above :)